administrator-documentation/moderne-platform/references/reporting.md

```diff
@@ -20,7 +26,7 @@ Audit logs are accessible from `https://<TENANT>.moderne.io/admin/audit-logs`, a
 
 To download audit logs, use the "Export to CEF" button: ![](<../../../.gitbook/assets/image (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1) (1).png>)
 
-## Other reports (recipe runs and commits)
+## Other reports (recipe runs, commits and usages)
```

Comment:
I know we are adding the report to DX but I wasn't certain if we also wanted to add it to the reports tab